Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -2690,6 +2690,7 @@ def gen2021HiMix(fragment,howMuch):
     '--secondfilein':'filelist:step1_dasparentquery.log',
     '--customise_commands':'"process.HLTAnalyzerEndpath.remove(process.hltL1TGlobalSummary)"',
     '--fileout':'DQMHLTonAOD.root',
+    '--procModifiers':'tau_readOldDiscriminatorFormat',
```

Comment:
IIUC, the problem shows up in DQMHLTonAOD_2017 as well (used in 136.7801).
This will address only 136.7803